middleCharacter.js

```diff
@@ -0,0 +1,10 @@
+// Get the Middle Character
+
+function getMiddle(value) {
+  if (value.length % 2 == 0)
+    return value[value.length / 2 - 1] + value[value.length / 2];
```

Comment:
please avoid using magic numbers also hold the middle value in a meaningful defined variable.